rali/rali_pybind/example/tf_classification_pipeline.py

```diff
@@ -33,13 +33,15 @@ def __init__(self, feature_key_map, tfrecordreader_type, batch_size, num_threads
 											image_type=types.RGB,
 											mean=[0.485 * 255,0.456 * 255,0.406 * 255],
 											std=[0.229 * 255,0.224 * 255,0.225 * 255])
+		self.one_hot_labels = ops.OneHot(num_classes=1000)
 		self.coin = ops.CoinFlip(probability=0.5)
 		print('rali "{0}" variant'.format(rali_device))
 
 	def define_graph(self):
 		inputs = self.input(name ="Reader")
 		images = inputs["image/encoded"]
 		labels = inputs["image/class/label"]
+		labels = self.one_hot_labels(labels)
```

Comment:
Looks like the code always gives one hot labels. Is this intended? I think it is better to have an option

Reply:
Added Option for One Hot labels